pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -54,7 +54,7 @@ def test_all(self):
         body = gen_distribution()
         body['publication'] = publication['_href']
         distribution = client.using_handler(api.task_handler).post(
-            DISTRIBUTION_PATH, body
+            BASE_DISTRIBUTION_PATH + 'rpm/rpm/', body
```

Comment:
I suggest you to create a `RPM_DISTRIBUTION_PATH` on tests/functional/constants like in
https://github.com/pulp/pulp_file/blob/master/pulp_file/tests/functional/constants.py#L21

Reply:
I agree looks better in the code. Updated.